frog/imports/ui/engine/QuestionDisplay.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component, PropTypes } from 'react';
+import ReactDOM from 'react-dom';
+import ReactPlayer from 'react-player';
+
+export default class QuestionDisplay extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      correct: -1,
+      answered: false,
+    }
+  }
+
+  checkQuestion(event){
+    event.preventDefault();
+    this.setState({
+      correct: this.state.answered ? this.state.correct : document.getElementById(this.props.question.concat(this.props.answer)).checked ? 1 : 0,
+      answered: true,
+    });
+  }
+
+  renderQuestionChoices(){
+    return (this.props.choices ?
+      this.props.choices.map((choice) =>
+        <span key={this.props.question.concat(choice)}>
+          <input
+            id = {this.props.question.concat(choice)}
+            type="radio"
+            name={this.props.question}
+            ref="quizChoices"
+            value={choice}
+          />{choice}<br/>
+        </span>) : <li>No Choices</li>);
+  }
+
+  renderCorrect(){
+    return (this.state.correct == 1 ? "Correct! The answer was ".concat(this.props.answer) : this.state.correct == 0 ? "Incorrect. The answer was ".concat(this.props.answer) : "Select an answer.");
+  }
+
+  render() {
+    return (
+      <div key={this.props.key}>
```

Comment:
That key is unnecessary as you are not creating a list of `<div>` for react to handle. Furthermore it raises an error to do `this.props.key` because the property `key` is reserved to be used by react, so you cannot use it ;)